analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -275,6 +271,11 @@ open class Npm(
                 }
             }
 
+            downloadUrl = downloadUrl
+                // Work around the issue described at
+                // https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285/19.
```

Comment:
Note: The mentioned URL is no longer reachable.

Reply:
Interesting. I cannot find any information about whether this host was deprecated... so maybe it's just a temporary glitch.